packages/react-atlas-docs/src/components/dialog/Dialog.js

```diff
@@ -0,0 +1,13 @@
+import React, { PropTypes } from "react";
+import * as Atlas from 'react-atlas';
+
+export default class Drawer extends React.Component {
+  constructor(props){
+    super(props)
+  }
+  render() {
+    return (
+      <Atlas.Drawer {...this.props}></Atlas.Drawer>
```

Comment:
this is interesting syntax.  Is this a way we are planning to handle some components going forward using the <Atlas.{component} notation?